src/core/stdc/tgmath.d

```diff
@@ -869,10 +869,23 @@ else
 
     ///
     alias core.stdc.math.fabs          fabs;
-    ///
-    alias core.stdc.math.fabsf         fabs;
-    ///
-    alias core.stdc.math.fabsl         fabs;
+    version( CRuntime_Microsoft )
+    {
+        extern(C++)
+        {
+            ///
+            float   fabs(float x);
+            ///
+            real    fabs(real x);        
```

Comment:
Makes sense. With the double being the c version, it covers both float and double